src/script/media/MediaConstraintsHandler.js

```diff
@@ -138,6 +146,17 @@ export default class MediaConstraintsHandler {
       return Promise.resolve(streamConstraints);
     }
 
+    if (!!navigator.mediaDevices.getDisplayMedia) {
```

Comment:
Since Firefox also implements this method, we could move that condition up (before the firefox condition) so that we go into the legacy `getUserMedia` only if the preferred methods are not available